--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -8,7 +8,7 @@
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
 
-find_package(Qt6 REQUIRED COMPONENTS Core Gui Widgets)
+find_package(Qt5 REQUIRED COMPONENTS Core Gui Widgets)
 find_package(SDL2 REQUIRED)
 
 # Prefer the regular target, but fall back to the static-only target so a
@@ -46,9 +46,9 @@
 
 target_link_libraries(QJoysticks
     PUBLIC
-        Qt6::Core
-        Qt6::Gui
-        Qt6::Widgets
+        Qt5::Core
+        Qt5::Gui
+        Qt5::Widgets
     PRIVATE
         ${QJ_SDL2_TARGET})
 
